src/ExtraRegisters.cc

```diff
@@ -144,6 +144,9 @@ static uint64_t xsave_features(const vector<uint8_t>& data) {
                                                   xsave_header_offset);
 }
 
+#if defined (__i386__)
+  typedef long double __uint128_t;
```

Comment:
This doesn't look right.

I think we can just replace `sizeof(__uint128_t)` with `16`.

Reply:
So it should read this:
```
        ((regno - DREG_V0) * 16/*sizeof(__uint128_t)*/), 16/*sizeof(__uint128_t)*/);
```
Or would be this better:
```
        ((regno - DREG_V0) * 16), 16); // type __uint128_t is not defined at i386.
```

Reply:
This:
```
((regno - DREG_V0) * 16), 16);
```